instrumentation/apache-dubbo-2.7/javaagent/build.gradle.kts

```diff
@@ -14,29 +14,9 @@ muzzle {
 dependencies {
   implementation(project(":instrumentation:apache-dubbo-2.7:library-autoconfigure"))
 
-  library("org.apache.dubbo:dubbo:2.7.0")
-}
-
-val latestDepTest = findProperty("testLatestDeps") as Boolean
+  library("org.apache.dubbo:dubbo:2.7.5")
```

Comment:
does the instrumentation no longer work against `2.7.0`? if so, can you add a comment here briefly explaining? thanks

Reply:
The testing module use `class DubboBootstrap` which is since 2.7.5, the ut will fail for 2.7.0 for classnotfound
so we only test it work pass (2.7.5,)

Reply:
it sounds like `2.7.0` may work for `library` (compilation) dependency, and you can bump the `testLibrary` version to `2.7.5`?